maven-enforcer-plugin/src/site/site.xml

```diff
@@ -39,7 +39,7 @@ under the License.
       <item name="Check specific rule via CLI" href="examples/check-specific-rule-via-cli.html"/>
     </menu>
     <menu name="Resources">
-      <item name="Version Ranges" href="https://cwiki.apache.org/confluence/display/MAVENOLD/Dependency+Mediation+and+Conflict+Resolution#DependencyMediationandConflictResolution-DependencyVersionRanges"/>
+      <item name="Version Ranges" href="../../enforcer/enforcer-rules/versionRanges.html"/>
```

Comment:
On other pages, e.g. https://maven.apache.org/enforcer/enforcer-rules/banDynamicVersions.html ,the "version ranges" links to: https://maven.apache.org/pom.html#Dependency_Version_Requirement_Specification

I suggest to bind all links to the same page.

Reply:
@Bukama I think it is ok